c_binding/include/libstoragemgmt/libstoragemgmt_plug_interface.h

```diff
@@ -1004,6 +1004,84 @@ typedef int (*lsm_plug_volume_ident_led_set) (lsm_plugin_ptr c,
 typedef int (*lsm_plug_volume_ident_led_clear) (lsm_plugin_ptr c,
                                                 lsm_volume * volume,
                                                 lsm_flag flags);
+/**
+ * Query the RAM cache information of a volume
+ * @param[in]   c               Valid lsm plug-in pointer
+ * @param[in]   volume          Volume to be deleted
+ * @param[out] write_cache_policy
+ *                         The write cache policy. Valid values are:
+ *                          * LSM_VOLUME_WRITE_CACHE_POLICY_WRITE_BACK
+ *                            The storage system will use write back mode if
+ *                            cache hardware found. In write back mode, I/O
+ *                            write data is done asynchronously to the cache and
+ *                            to the backing store(normally disk), the
+ *                            acknowledgement will send back to I/O initiator
```

Comment:
Change to "the acknowledgement will be sent back to the I/O initiator once the data has been written to the cache".